contrib/postgres_fdw/expected/mpp_postgres_fdw.out

```diff
@@ -207,8 +210,139 @@ explain (costs off) select * from fs1, fs2 where fs1.a = fs2.a and fs1.gp_foreig
 select * from fs1,fs2 where fs1.a = fs2.a and fs1.gp_foreign_server = fs2.gp_foreign_server;
  a |  b   | a |  b   
 ---+------+---+------
+ 1 | fdw3 | 1 | fdw3
+ 1 | fdw1 | 1 | fdw1
  1 | fdw2 | 1 | fdw2
+(3 rows)
+
+explain (costs off) select count(*) from fs1, fs2 where fs1.a = fs2.a;
+                                  QUERY PLAN                                  
+------------------------------------------------------------------------------
+ Finalize Aggregate
+   ->  Gather Motion 3:1  (slice1; segments: 3)
+         ->  Partial Aggregate
+               ->  Hash Join
+                     Hash Cond: (fs1.a = fs2.a)
+                     ->  Redistribute Motion 3:3  (slice2; segments: 3)
+                           Hash Key: fs1.a
+                           ->  Foreign Scan on fs1
+                     ->  Hash
+                           ->  Redistribute Motion 3:3  (slice3; segments: 3)
+                                 Hash Key: fs2.a
+                                 ->  Foreign Scan on fs2
+ Optimizer: Postgres query optimizer
+(13 rows)
+
+select count(*) from fs1,fs2 where fs1.a = fs2.a;
+ count 
+-------
+     9
+(1 row)
+
+explain (costs off) select count(*) from fs1, fs2 where fs1.a = fs2.a and fs1.gp_foreign_server = fs2.gp_foreign_server;
+                           QUERY PLAN                           
+----------------------------------------------------------------
+ Finalize Aggregate
+   ->  Gather Motion 3:1  (slice1; segments: 3)
+         ->  Foreign Scan
+               Relations: Aggregate on ((fs1) INNER JOIN (fs2))
+ Optimizer: Postgres query optimizer
+(5 rows)
+
+select count(*) from fs1,fs2 where fs1.a = fs2.a and fs1.gp_foreign_server = fs2.gp_foreign_server;
+ count 
+-------
+     3
+(1 row)
+
+----------------------------
+-- Test with enable parallel
+----------------------------
+set enable_parallel to true;
+explain (costs off) select * from fs1;
```

Comment:
fdw can't parallel ? I see still `Gather Motion 3:1 ` ?

Reply:
Yeah, I think it is.